### DISCUSSION TITLE
Problems adding issues/PRs to a project using search box

### DISCUSSION THREAD

**Discussion by james-skinner-deltatre** [community/community#13890] [Projects and Issues]
Specifically I mean this search box:
![image](https://user-images.githubusercontent.com/74296503/160867017-4eb53838-0f40-483a-99fc-7784dddf22ae.png)

And I am talking about a private project with private repositories in an organisation

My problems are:
1. I can't search for just an issue or PR title in my organisation, has to be the repo selection first
2. If a repo has issues disabled, it does not show in the list, so I cannot add a pull request to the project this way
3. Its pretty slow
4. The repo filtering does not seem to work very well:

Here I am trying to find `axis-svc-search` repo:

nope:
![image](https://user-images.githubusercontent.com/74296503/160868073-d82ac876-fcb1-496f-b331-f1146bc3b568.png)
nope:
![image](https://user-images.githubusercontent.com/74296503/160868140-9aef1c99-40e4-4220-9f21-07d34da8d2d5.png)
nope:
![image](https://user-images.githubusercontent.com/74296503/160868433-4fa69300-4dce-4177-82d6-236402d49c5c.png)

yep:
![image](https://user-images.githubusercontent.com/74296503/160868287-aa3c8f67-df1e-40a8-a70d-0bbec43cd9e1.png)
yep:
![image](https://user-images.githubusercontent.com/74296503/160868360-342597b3-d27d-4ea1-aedc-1130e3d17767.png)

**Comment by ggdouglas** [comment]
> If a repo has issues disabled, it does not show in the list, so I cannot add a pull request to the project this way

Thank you! I couldn't for the life of me figure out why one of the repos that I wanted to reference in a project wasn't available in search. Turns out that I had issues disabled on that particular repo. Not very intuitive!
